### PR TITLE
(GH-2044) Updated readme to include Chocolatey package

### LINKED ISSUE #2044: Add chocolatey installation method
Hi,

I have created an @chocolatey [package](https://chocolatey.org/packages/microsoft-windows-terminal) for Windows Terminal. Are you willing to add this installation method to your documentation?

Thanks,

Maurice

### PR DESCRIPTION
**Pull request by mkevenaar**
## Summary of the Pull Request
Updated documentation regarding the Chocolatey package installation 

## References
#2044

## PR Checklist
* [X] Closes #2044 
* [X] CLA signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/Terminal) and sign the CLA
* [X] Tests added/passed
* [X] Requires documentation to be updated
* [X] I've discussed this with core contributors already. If not checked, I'm ready to accept this work might be rejected in favor of a different grand plan. Issue number where discussion took place: #2044 

## Detailed Description of the Pull Request / Additional comments
Updated documentation

## Validation Steps Performed
N/A

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,13 +10,30 @@
...
 
 ### Microsoft Store
 
 Download the Microsoft Terminal free from the Microsoft Store and it'll be continuously updated. Or, feel free to side-load [releases](https://github.com/microsoft/terminal/releases) from GitHub, but note they won't auto-update.
 
 <a href='//www.microsoft.com/store/apps/9n0dx20hk701?cid=storebadge&ocid=badge'><img src='https://assets.windowsphone.com/85864462-9c82-451e-9355-a3d5f874397a/English_get-it-from-MS_InvariantCulture_Default.png' alt='English badge' width="284" height="104" style='width: 284px; height: 104px;'/></a>
 
+### Chocolatey (Unofficial)
+
+Download and update Microsoft Terminal from [Chocolatey](https://chocolatey.org)
```
Comment (gep13) [inline README.md]:
```suggestion
Download and upgrade Microsoft Terminal from [Chocolatey](https://chocolatey.org).
```

File: README.md [hunk 2]
```diff
@@ -10,13 +10,30 @@
...
+```powershell
+choco install microsoft-windows-terminal
+```
+
+To upgrade Windows Terminal, run the following command from the command line or from PowerShell:
+```powershell
+choco upgrade microsoft-windows-terminal
+```
+
+If you have any issues when installing the package please go to the [package page](https://chocolatey.org/packages/microsoft-windows-terminal) and follow the [Chocolatey triage process](https://chocolatey.org/docs/package-triage-process)
```
Comment (gep13) [inline README.md]:
```suggestion
If you have any issues when installing/upgrading the package please go to the [package page](https://chocolatey.org/packages/microsoft-windows-terminal) and follow the [Chocolatey triage process](https://chocolatey.org/docs/package-triage-process)
```

**Comment by mkevenaar** [comment]
Let me fix that real quick.

**Comment by mkevenaar** [comment]
@DHowett-MSFT I don't know how this PR could break the build. Could you please verify?

File: README.md [hunk 3]
```diff
@@ -10,13 +10,30 @@
...
 
 ### Microsoft Store
 
 Download the Microsoft Terminal free from the Microsoft Store and it'll be continuously updated. Or, feel free to side-load [releases](https://github.com/microsoft/terminal/releases) from GitHub, but note they won't auto-update.
 
 <a href='//www.microsoft.com/store/apps/9n0dx20hk701?cid=storebadge&ocid=badge'><img src='https://assets.windowsphone.com/85864462-9c82-451e-9355-a3d5f874397a/English_get-it-from-MS_InvariantCulture_Default.png' alt='English badge' width="284" height="104" style='width: 284px; height: 104px;'/></a>
 
+### Chocolatey (Unofficial)
+
+Download and upgrade Microsoft Terminal from [Chocolatey](https://chocolatey.org).
```
Comment (zadjii-msft) [inline README.md]:
```suggestion
Download and upgrade the Windows Terminal from [Chocolatey](https://chocolatey.org).
```

**Comment by zadjii-msft** [comment]
Oh you didn't break the build. We've been running into some infrastructure issues with our build lab, so I wouldn't be worried about that.

**Comment by mkevenaar** [comment]
@zadjii-msft I have committed your suggestion.

**Comment by miniksa** [comment]
> Oh you didn't break the build. We've been running into some infrastructure issues with our build lab, so I wouldn't be worried about that.

I'm correcting that in #2096 to unblock everyone.

**Comment by mkevenaar** [comment]
@miniksa Thank you!

**Pull request merged**